### ISSUE TITLE
ToggleButton is shown as a dot during program execution.

### ISSUE THREAD

**Issue by rogamaral** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#365]
**TvSerieView.xaml:**
Design view:

![tvserieview](https://cloud.githubusercontent.com/assets/5101967/14959479/7405b830-1087-11e6-95db-11c86a3ffebb.png)

xaml code:

```
<UserControl x:Class="MySeriesTracker.View.TvSerieView"
             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
             xmlns:materialDesign="http://materialdesigninxaml.net/winfx/xaml/themes"
             xmlns:viewModel="clr-namespace:Cliente.Mvvm.ViewModel;assembly=Cliente.Mvvm"
             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
             mc:Ignorable="d" 
             d:DesignWidth="400"
             d:DataContext="{d:DesignInstance Type=viewModel:TvSerieViewModel, IsDesignTimeCreatable=True}">

    <UserControl.Resources>
        <ResourceDictionary>
            <ResourceDictionary.MergedDictionaries>
                <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/materialdesigntheme.togglebutton.xaml" />
            </ResourceDictionary.MergedDictionaries>
        </ResourceDictionary>
    </UserControl.Resources>

    <Grid DataContext="{Binding Path=TvSerie}">
        <Grid.RowDefinitions>
            <RowDefinition Height="Auto"/>
        </Grid.RowDefinitions>
        <Grid.ColumnDefinitions>
            <ColumnDefinition Width="Auto"/>
            <ColumnDefinition Width="*"/>
            <ColumnDefinition Width="Auto"/>
        </Grid.ColumnDefinitions>
        <Image Grid.Column="0" Height="100" Width="178" Source="{Binding ImagePath}"/>

        <StackPanel Grid.Column="1">
            <TextBlock Margin="8" FontSize="18" FontWeight="DemiBold"
                       Text="{Binding MasterTitle}" />
        </StackPanel>
            <ToggleButton Grid.Column="2" Style="{StaticResource MaterialDesignActionAccentToggleButton}"
                          IsChecked="{Binding IsFollowed, Mode=TwoWay}" >
                      <materialDesign:PackIcon Kind="Plus"/>
            </ToggleButton>
    </Grid>
</UserControl>

```

**MainWindow.xaml:**
Design view:

![mainwindow](https://cloud.githubusercontent.com/assets/5101967/14959877/8d522fec-1089-11e6-80ab-d17f1c0aee85.png)

xaml code:

```
<controls:MetroWindow x:Class="MySeriesTracker.View.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:controls="http://metro.mahapps.com/winfx/xaml/controls"
        xmlns:viewModel="clr-namespace:Cliente.Mvvm.ViewModel;assembly=Cliente.Mvvm"
        xmlns:views="clr-namespace:MySeriesTracker.View"

        TextElement.Foreground="{DynamicResource MaterialDesignBody}"
        Background="{DynamicResource MaterialDesignPaper}"                    
        TextElement.FontWeight="Medium"
        TextElement.FontSize="14"
        FontFamily="{StaticResource DefaultFont}"

        DataContext="{Binding Main, Source={StaticResource Locator}}"
        WindowStartupLocation="CenterScreen"
        Title="MySeriesTracker" TitleCaps="False" Height="350" Width="525">

    <controls:MetroWindow.Resources>
        <ResourceDictionary>
            <DataTemplate DataType="{x:Type viewModel:TvSerieViewModel}">
                <views:TvSerieView/>
            </DataTemplate>
        </ResourceDictionary>
    </controls:MetroWindow.Resources>

    <Grid>
        <Grid.RowDefinitions>
            <RowDefinition Height="Auto"/>
        </Grid.RowDefinitions>
        <Grid.ColumnDefinitions>
            <ColumnDefinition Width="*"/>
        </Grid.ColumnDefinitions>
        <ContentControl Content="{Binding ActualViewModel}"/>
    </Grid>
</controls:MetroWindow>

```

Running view:

![running](https://cloud.githubusercontent.com/assets/5101967/14960265/c66a44c0-108b-11e6-9cc0-285ea3d73172.png)

App.xaml copied from MahMaterialDragablzMashUp example, without the Dragablz stuff.
MahApps.Metro.1.2.0.0
MaterialDesignColors.1.1.2
MaterialDesignThemes.1.5.0.523
MaterialDesignThemes.MahApps.0.0.4

**Comment by ButchersBoy** [comment]
I think this bug is fixed in 2.0.  Can you try the -pre nuget package?

**Comment by rogamaral** [comment]
Yes, I will try in a copy of my solution. At what stage is the version 2? When is the stable version ready?

**Comment by ButchersBoy** [comment]
Pretty stable.  No cast-iron release dates but I try and release quite regularly.  Quite a few people are on the pre-release, so it gets tested quite well.

See:

https://github.com/ButchersBoy/MaterialDesignInXamlToolkit/issues?utf8=%E2%9C%93&q=is%3Aissue+milestone%3A%22ver+2.0.0%22+

https://github.com/ButchersBoy/MaterialDesignInXamlToolkit/wiki/Version-2.0.0---Breaking-Changes

**Comment by rogamaral** [comment]
The problem seems to be fixed in 2.0 but now the check mark Icon(onContent) is not shown during design.

![designv2](https://cloud.githubusercontent.com/assets/5101967/14961978/7cce041a-1094-11e6-91c9-16d304521c43.png)

offContent is OK.

**Comment by ButchersBoy** [comment]
and at runtime?

**Comment by rogamaral** [comment]
At run time everything works fine.

**Comment by Keboo** [comment]
I believe the issue with the PackIcon at design time is now fixed in the latest 3.0.0 preview NuGet packages
